presto-tests/src/main/java/com/facebook/presto/tests/H2QueryRunner.java

```diff
@@ -404,6 +404,12 @@ else if (type instanceof RowType) {
                     .toArray();
         }
 
+        if (elementType instanceof CharType) {
+            return Arrays.stream(values)
+                    .map(v -> v == null ? null : padEnd((String) v, ((CharType) elementType).getLength(), ' '))
```

Comment:
nit: consider extracting length variable and use `.map(String.class::cast)` for readability:

```
            int length = ((CharType) elementType).getLength();
            return Arrays.stream(values)
                    .map(String.class::cast)
                    .map(v -> v == null ? null : padEnd(v, length, ' '))
                    .toArray();
```